src/main.rs

```diff
@@ -170,3 +172,25 @@ fn main() {
         Err(err) => error(err.description()),
     }
 }
+
+fn transform_args_with_exec() -> Vec<OsString> {
```

Comment:
Could we add a doc-string here to describe what the function does?

Reply:
For sure, no problem